src/automaton/core/network/BUILD

```diff
@@ -67,3 +67,25 @@ cc_library(
   }),
   linkstatic=True,
 )
+
+
+cc_library(
+  name = "rpc_server",
+  srcs = [
+    "rpc.h",
+    "rpc.cc",
+  ],
+  deps = [
+    "@localboost//:asio",
+    "//automaton/core/io",
+  ],
+  linkopts = select({
+    "//conditions:linux": ["-lpthread",],
```

Comment:
This we may want to consider to put in the localboost asio BUILD rules. Because that's what causes it (I think)